### ISSUE TITLE
Desktop icon should be at least 64x64 pixel

### ISSUE THREAD

**Issue by apoleon** [fbergo/eboard#6]
The best icon is currently icon32.png which is a 32x32 sized logo. For best user experience I suggest to ship an 64x64 icon at least, bigger icon sizes are preferred nowadays, up to 256x256 or even a svg icon would be great.

Those icons should be installed to /usr/share/icons/hicolor/${size}/apps instead of /usr/share/pixmaps. This location has been deprecated.

**Comment by fbergo** [comment]
I'm looking into this and will add SVG and higher resolution icons soon. Regarding the install location: I am a Fedora/CentOS user and I am not familiar with Debian/Ubuntu packaging. The install target on the Makefile does not install any icon, nor does the eboard.spec file used for RPM building. It appears that the issue with the install location is in your packaging specification (which is not part of this git repository).

**Comment by apoleon** [comment]
Not quite. This is a freedesktop.org specification.

https://standards.freedesktop.org/desktop-entry-spec/latest/

Red Hat / Fedora and all other distributions already install .desktop
files to /usr/share/applications, appstream files to /usr/share/metainfo
and icons to /usr/share/icons/hicolor

It is true that your build system does not take care of integrating your
game into the user's desktop but it would be great if it did. The icon
is only the part that most people recognize immediately because it
doesn't look very good on modern desktop environments like Gnome/KDE.

**Comment by fbergo** [comment]
I'll add new icons, fix any memory bugs and ensure it compiles with current compilers, but I wrote eboard around 2002-2008 and I even I no longer use it regularly. It is based on a deprecated version of GTK and I have no intention to devote much time to maintaining it, keeping it current or porting to a newer toolkit.

**Comment by apoleon** [comment]
Fair enough. The memory bugs are the most important ones because they prevent users from playing. The Debian package didn't receive much love in the past, so I just wanted to forward everything to you that other users reported in the past ten years. 

https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=eboard
https://tracker.debian.org/pkg/eboard
